tests/grant.test.ts

```diff
@@ -155,7 +155,43 @@ export function runTests(): void {
 		assert.assertTrue(gUpdate!.title != g!.title)
 		assert.assertTrue(gUpdate!.details != g!.details)
 
-		assert.assertTrue(gUpdate!.fields.includes(`${gUpdate!.id}.applicantName2`))
+		// assert.assertTrue(gUpdate!.fields.includes(`${gUpdate!.id}.applicantName2`))
+		const reward = Reward.load(g!.id)
+		assert.assertNull(reward?.token)
+	})
+
+	test('should update grant reward with token', () => {
+		const g = createGrant()
+
+		const ev = newMockEvent()
+
+		ev.parameters = [
+			new ethereum.EventParam('workspaceId', ethereum.Value.fromI32( 0x03 )),
+			new ethereum.EventParam('metadataHash', ethereum.Value.fromString( UPDATE_JSON_WITH_TOKEN )),
+			new ethereum.EventParam('active', ethereum.Value.fromBoolean(false)),
+			// the IPFS hash contains mock data for the workspace
+			new ethereum.EventParam('time', ethereum.Value.fromI32(130)),
+		]
+		ev.transaction.to = MOCK_GRANT_ID
+
+		const event = new GrantUpdated(ev.address, ev.logIndex, ev.transactionLogIndex, ev.logType, ev.block, ev.transaction, ev.parameters)
+		handleGrantUpdated(event)
+
+		const gUpdate = Grant.load(g!.id)
+		assert.i32Equals(gUpdate!.updatedAtS, 130)
+		assert.booleanEquals(gUpdate!.acceptingApplications, false)
+		assert.stringEquals(gUpdate!.workspace, BigInt.fromI32(0x03).toHex())
+
+		assertStringNotEmpty(gUpdate!.title)
+		assertStringNotEmpty(gUpdate!.summary)
+
+		assert.assertTrue(gUpdate!.summary != g!.summary)
+		assert.assertTrue(gUpdate!.title != g!.title)
+		assert.assertTrue(gUpdate!.details != g!.details)
+
+		// assert.assertTrue(gUpdate!.fields.includes(`${gUpdate!.id}.applicantName2`))
+		const reward = Reward.load(g!.id)
+		assert.assertNotNull(reward?.token)
```

Comment:
Shouldn't this be asserted null?

Reply:
We are passing the token with the reward in this test. So we want the `reward.token` to have some value